driver-core/src/main/java/com/datastax/driver/core/querybuilder/Delete.java

```diff
@@ -114,6 +132,7 @@ public Where where() {
      * @return the conditions of this query to which more conditions can be added.
      */
     public Conditions onlyIf(Clause condition) {
```

Comment:
Could you add in this class the same javadoc warnings about idempotence as in `Insert` and `Update`?